README.md

```diff
@@ -5,11 +5,11 @@ Very simple Roles library without any authorization enforcement supporting scope
 Let's see an example:
 
 ```ruby
-user.has_role?(:moderator, Forum.first)
+user.has_role?(:moderator, @forum)
```

Comment:
A forum instance gives you one less concept to think about when trying to understand the example.